pkg/fuzzer/fuzzer.go

```diff
@@ -102,8 +101,8 @@ type SignalType int
 
 const (
 	NoSignal  SignalType = 0 // we don't need any signal
-	NewSignal SignalType = 1 // we need the newly seen signal
-	AllSignal SignalType = 2 // we need all signal
+	NewSignal                // we need the newly seen signal
+	AllSignal                // we need all signal
```

Comment:
Now they all have the same value?

Reply:
Good point, I read it as "=iota". Fixed.